public/app/plugins/datasource/mssql/sqlCompletionProvider.ts

```diff
@@ -35,13 +36,17 @@ export const getSqlCompletionProvider: (args: CompletionProviderGetterArgs) => L
           processedToken = processedToken.next;
         }
 
-        return tablePath;
+        return { table: tablePath };
       },
     },
 
     columns: {
-      resolve: async (t: string) => {
-        const [database, schema, tableName] = t.split('.');
+      resolve: async (t: TableIdentifier | undefined) => {
```

Comment:
This seems to be the only change in the backport that changes code beyond an import. I don't know enough about the mssql datasource but should these changes be backported?

Reply:
@scottlepp WDYT?

Reply:
This should be okay.